### PR TITLE
Improved journey for damp, leak, mould

### PR DESCRIPTION
**Pull request by deberny**
Resolves trello card
https://trello.com/c/ulPBgzNt/75-ko-journey-for-damp-vs-leak

### CODE REVIEW

**Comment by deberny** [comment]
@james I've kept the old damp leak page in case we need to use it.

This is the new journey. There will be need to add meaningful free text description depending on the problem a user ends up with. Amy is working on it.

**Comment by deberny** [comment]
[KO journey for Damp VS Leak](https://trello.com/c/ulPBgzNt/75-ko-journey-for-damp-vs-leak)

**Review by james — APPROVED** [review]
I'm not sure if you want to wait for the description bits before merging, so I'll leave that to you, but LGTME!

**Pull request merged**
